src/beacon-api/app/routes/sites.py

```
@@ -275,59 +314,123 @@
     return {
-        "count": len(regions),
-        "regions": sorted(regions)
+        "count": len(cities),
+        "cities": sorted(cities)
     }
 
 
 @router.get("/districts/list")
 async def get_districts(
     *,
     db: Session = Depends(get_db),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)